pkg/http/analytics.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2022 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/DopplerHQ/cli/pkg/utils"
+)
+
+func CaptureCommand(command string) ([]byte, Error) {
+	postBody := map[string]interface{}{"command": command}
+	body, err := json.Marshal(postBody)
```

Comment:
Is there anything else we might want to capture that isn't intrusive?
Ideas:
- True/false: is `api-host` set to `https://api.doppler.com`?
- True/false: is `token` set?
- True/false: is the `scope` set to `/`?

Reply:
The scope one is particularly interesting. I have wondered how frequently users are specifying a scope rather than using the default global scope (specifically when logging in and when running setup). I'll create a separate ticket for this.

Reply:
Created ENG-5666